model/DataObject.php

```diff
@@ -1717,7 +1717,8 @@ public function getRemoteJoinField($component, $type = 'has_many', &$polymorphic
 	 * @todo Implement query-params
 	 */
 	public function getManyManyComponents($componentName, $filter = "", $sort = "", $join = "", $limit = "") {
```

Comment:
I agree with that proposal, perhaps one for a separate PR - we’ve already expanded this one a fair bit! :)